project.js

```diff
@@ -599,6 +613,10 @@ function render() {
     for (const obj of Object.values(objects)) {
         if (obj['isRendering']) renderObject(obj);
     }
+    let time = getTimeAngle();
+    
+    stationaryLightPosition = vec4(circlePoints[time % circlePoints.length],circlePoints[time % circlePoints.length], -100.0, 1);
```

Comment:
Good start, just make sure the scene objects are well lit at certain times. Currently, there's no time that has a better illumination than the previously use constant